### PR TITLE
Add new "Lavender" theme

### PR DESCRIPTION
**Pull request by Osyx**
Added a new purple theme.

### Tested
- OnePlus 8 Pro hardware with Android 12
- Both light and dark mode, different views, dialogues etc.

### Images
| Light | Dark |
| ------- | ------- |
|  | dark theme preview |
|  | image |
|  | image |
|  | image |

### CODE REVIEW

**Comment by jferdous70** [comment]
I would prefer it be called Lavender

**Comment by Osyx** [comment]
> I would prefer it be called Lavender

Sure, works for me, I'll wait and see if there's any further feedback before changing though

**Comment by arkon** [comment]
Is this meant to address #6430? As-is, it looks a bit too much like a gold-less Tako theme.

**Comment by Osyx** [comment]
> Is this meant to address #6430? As-is, it looks a bit too much like a gold-less Tako theme.

Isn't the gold part the main factor of the Tako theme? But sure I could change it up to better address that issue instead!

**Comment by arkon** [comment]
Tako is purple + gold.

**Comment by Osyx** [comment]
@arkon hmm why do Library > Display use a different color than Library > Display for the interactive elements? (checkboxes and radio). 
Display seems to use the primary color, while Filter seems to use the secondary

**Comment by arkon** [comment]
They should be consistent, are you seeing otherwise?

**Comment by Osyx** [comment]
Yes, saw it now while trying the new theme style
| Filter | Display |
| ------- | ------- |
| image | image |

**Comment by Osyx** [comment]
Is this better for https://github.com/tachiyomiorg/tachiyomi/issues/6430?
| Light | Dark |
| ------- | ------- |
| image | image |

**Comment by arkon** [comment]
> The Blue checkboxes are using the secondary color while the purple boxes are using the primary color.

Fixed in cd5bcc36734e0959569be6dd5ecd3b3813863207

**Comment by Osyx** [comment]
Updated images in description.

@arkon does this look ok now?

**Comment by arkon** [comment]
I'd actually like to see some middleground between this and #7350:

- For light theme, I like that #7350 is more purple (matches more of what I'd consider "lavender"), although I feel like it could be a bit lighter
- For dark theme, #7350 seems too dark.

Maybe you and @CrepeTF can sync up and figure out some combination? I don't want to add 2 purple themes, since they overlap too much.

**Comment by Osyx** [comment]
Sure, that works, @CrepeTF how do you want to sync? Discord or mail works for me, if you want to talk outside of GitHub.

**Comment by CrepeTF** [comment]
@Osyx I'm primarily on Discord, so just @ me in the Tachiyomi server

File: app/src/main/java/eu/kanade/tachiyomi/widget/materialdialogs/QuadStateTextView.kt [hunk 1]
```diff
@@ -29,7 +29,7 @@ class QuadStateTextView @JvmOverloads constructor(context: Context, attrs: Attri
         val tint = if (state == State.UNCHECKED) {
             context.getThemeColor(R.attr.colorControlNormal)
         } else {
-            context.getThemeColor(R.attr.colorAccent)
+            context.getThemeColor(R.attr.colorPrimary)
```
Comment (Osyx) [inline app/src/main/java/eu/kanade/tachiyomi/widget/materialdialogs/QuadStateTextView.kt:32]: @arkon is this alright? Seems to fix it for Set categories sheet at least. Idk if you rather me do another PR or if you just want to cherry-pick or something.
| Before | Now |
| ------ | ----- |
| image | image |

Comment (Osyx) [reply app/src/main/java/eu/kanade/tachiyomi/widget/materialdialogs/QuadStateTextView.kt:32]: Or maybe it's just me having weird theme conf with the secondary colour?

Comment (arkon) [reply app/src/main/java/eu/kanade/tachiyomi/widget/materialdialogs/QuadStateTextView.kt:32]: Should be fine here.

**Comment by Osyx** [comment]
Should be finished now, but can't add screenshots of the light theme @CrepeTF fixed until https://github.com/tachiyomiorg/tachiyomi/issues/7368 is fixed.
